demo/kserve/install-manual.md

```diff
@@ -1,13 +1,28 @@
-# Manual Installation
-## Export Environment Variables
-Set the variables below with desired values.
-`TARGET_OPERATOR` could be either `odh` or `rhods`.
-~~~
-export TARGET_OPERATOR=odh
-~~~
-## Clone the repo and setup the environment
+# Step-by-step command installation of Kserve and dependencies
+
+Note: You have the alternative option of installing the KServe/Caikit/TGIS stack by using [Script-based installation](/demo/kserve/scripts/README.md).
+
+
+## Set the data science operator
+
+Set the `TARGET_OPERATOR` environment variable to either `odh`  or `rhods`.
+
```

Comment:
In the demo/kserve/scripts/README.md instructions, they also have brew as an option. Is brew not applicable here?

Reply:
@MelissaFlinn I developed this script for all scenarios but `BREW` is only for internal Openshift AI developers. For this reason, we don't need to explain this option much to users.

Reply:
OK - I removed references to brew - thanks